docs/admin_api/user_admin_api.md

```diff
@@ -564,6 +564,11 @@ The following fields are returned in the JSON response body:
 
 ### List media uploaded by a user
 Gets a list of all local media that a specific `user_id` has created.
+These are media that the user has uploaded himself
+([local media](../media_repository.md#local-media)), as well as
+[preview images](../media_repository.md#url-previews) if the
```

Comment:
Just to help make it a bit more clear that it's not the user that sent a URL, but those who attempted to preview it.

```suggestion
[URL preview images](../media_repository.md#url-previews) requested by the user if the
```